src/blocks/youtube.v1.yaml

```diff
@@ -1,22 +1,24 @@
 $schema: http://json-schema.org/draft-04/schema#
 title: YouTube video
 type: object
-properties:
-    type:
-        type: string
-        enum:
-          - youtube
-    id:
-        title: Video ID
-        type: string
-    width:
-        title: Width
-        type: integer
-        minimum: 1
-    height:
-        title: Height
-        type: integer
-        minimum: 1
+allOf:
+  - $ref: ../misc/caption.v1.yaml
+  - properties:
+        type:
+            type: string
+            enum:
+              - youtube
+        id:
```

Comment:
This now conflicts with the `id` property inside `caption.v1.yaml` (which is meant for the HTML ID).

I'm wondering whether we should forget YouTube as a specific type, and just use an `embed` block type instead (which allows for arbitrary things to be used). That's a more complicated change though.

Reply:
This is no longer a conflict since the `id` property has been removed from `caption.v1.yaml` and must be referenced separately.